product.1.html

```diff
@@ -6,7 +6,7 @@
 		<meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
Please follow proper naming conventions for html files. i.e product related html files.